.github/workflows/partial-backend.yml

```
@@ -82,5 +88,17 @@ jobs:
         env:
           DB_ENGINE: ${{ matrix.Database }}
           POSTGRES_SERVER: localhost
+          LDAP_AUTH_ENABLED: True
+          LDAP_SERVER_URL: ldap://localhost:10389
```

```suggestion
          LDAP_SERVER_URL: ldap://ldap-service:10389
```

I think you can set it to the name of the service.

---

The reason I used localhost instead of the service name is mainly to keep it consistent with how the Postgres service is being used above